### ISSUE TITLE
Custom error messages issue

### ISSUE THREAD

**Issue by rverrastro** [rizwan3d/ValidationMyPhp#13]
In this version, the error custom messages doesn't work and always printed with default message.

The problem is in file Validation.php at line 48:
` $messages[$field][$rule_name] ?? $rule->message,`

Fixed changing with:
`$messages[$rule_name] ?? $rule->message,`

**Comment by rverrastro** [comment]
Hello @rizwan3d , do you have fixed it?

**Comment by rizwan3d** [comment]
Fixed

**Comment by roccoverrastro** [comment]
Hello @rizwan3d , after your fix on Jul 4 (version 0.0.5), the "unique" message doesn't work. Here an example:

```
        $errors = $this->validation->validate($data, [
            'email' => 'required|email|unique:invitations,email',
        ], [
            'required' => 'FIELD-REQUIRED',
            'email' => 'INVALID-EMAIL-ADDRESS',
            'unique' => 'EMAIL-EXISTS'
        ]);
```

If the email exists in the table "invitations", it always return "INVALID-EMAIL-ADDRESS" and now return it as array of array.

```
Array
(
  [email] => Array
  (
  [0] => INVALID-EMAIL-ADDRESS
  )
)
```
